api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -59,6 +78,20 @@ public BillingProjectBufferService(
     this.clock = clock;
     this.fireCloudService = fireCloudService;
     this.workbenchConfigProvider = workbenchConfigProvider;
+    View view = View.create(
+        View.Name.create("billing_project_buffer_entries"),
+        "The number of billing project buffer entries.",
```

Comment:
Does StackDriver distinguish between a counter and a gauge? We'd want this to be a gauge.

Reply:
See https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors#MetricKind

Reply:
Because we're using opencensus, this is probably the link that matters:
https://cloud.google.com/monitoring/custom-metrics/open-census (found it though.)